src/components/send/SendAssetList.js

```diff
@@ -186,7 +186,7 @@ export default class SendAssetList extends React.Component {
     collectibles.map(collectible => (
       <CollectiblesSendRow
         item={collectible}
-        key={collectible.id}
+        key={collectible.uniqueId}
```

Comment:
makes more sense to use the uniqueId here, since id was the tokenId which can overlap easily.